practitioner-service/src/main/resources/bootstrap.properties

```diff
@@ -0,0 +1,10 @@
+spring.application.name=practitioner-service
+
+spring.cloud.config.uri=http://localhost:8888
+
+spring.cloud.config.profile=default
+spring.cloud.config.label=main
+
+
```

Comment:
redundant empty lines

Reply:
not sure why this matters as its the end of the file? It would matter more if it was in a file that had further code as readability would be effected but this is the end of the file?

Reply:
doesnt matter. just a minor style comment